.gitignore

```diff
@@ -1 +1,2 @@
+Gemfile.lock
```

Comment:
You are supposed to check the Gemfile.lock in. This ensures that everyone gets the exact same version of the gems.